app/models/service_ansible_tower.rb

```diff
@@ -0,0 +1,26 @@
+class ServiceAnsibleTower < Service
+  include ServiceConfigurationMixin
+
+  alias_method :job_template, :configuration_script
+  alias_method :job_template=, :configuration_script=
+
+  def launch_job
+    @job = ManageIQ::Providers::AnsibleTower::ConfigurationManager::Job.create_job(job_template, {})
```

Comment:
An AnsibleTower service always has a job_template. It is added when the service template is defined.